tests/parallel.py

```diff
@@ -15,17 +15,28 @@
 """Unittests for Parallel Layer."""
 import os
 import unittest
-import numpy as np
+
 import jax.numpy as jn
+import numpy as np
 
 import objax
-
+from objax.typing import JaxArray
 
 # Split CPU cores into 8 devices for tests of objax.Parallel
+
 os.environ['XLA_FLAGS'] = ' '.join(os.environ.get('XLA_FLAGS', '').split(' ')
                                    + ['--xla_force_host_platform_device_count=8'])
 
 
+class LinearArgs(objax.nn.Linear):
```

Comment:
Seems like `LinearArgs` is not used in `parallel.py`

Reply:
I changed this file accidentally. Removing all changes.